exercises/book-store/src/example/java/BookStore.java

```diff
@@ -0,0 +1,76 @@
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class BookStore{
+
+	private int bookPrice, maxGroupSize;
+	private double discountIncrement;
+	private List<Integer> books;	
+
+	public BookStore (List<Integer> books){
+		this.books = books;
+		this.bookPrice = 8;
+	}	
+
+	public double CalculateTotalCost(){
+		return CalculateTotalCost(this.books,0);
+	}
+
+	private double CalculateTotalCost (List<Integer> books,double priceSoFar ){
+		double minPrice = Double.MAX_VALUE;
+
+
+		if(books.size() == 0)
```

Comment:
Stylistically we would prefer if you use braces, even for simple cases like this.  EG:
```Java
if (books.size() ==0){
    return priceSoFar;
}
```